Source/MQTTnet/Client/Options/MqttClientOptionsBuilderTlsParameters.cs

```diff
@@ -19,7 +19,11 @@ public class MqttClientOptionsBuilderTlsParameters
 
         public Func<MqttClientCertificateValidationCallbackContext, bool> CertificateValidationHandler { get; set; }
 
+#if NETCOREAPP3_0 || NET5_0
```

Comment:
Will NETCOREAPP3_0 work together with the nuget package? There is only 3.1 used. Shall we also move over to 3.1 instead?

    <!-- .NET Standard 3.1 -->
    <file src="..\Source\MQTTnet\bin\Release\netcoreapp3.1\MQTTnet.*" target="lib\netcoreapp3.1\"/>

Reply:
I have used NETCOREAPP3_0 according to [preprocessor-if](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives/preprocessor-if) but i think we can move over to 3.1 instead.